F1/F1/Constructor/constructorTableViewCell.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  ConstructorTableViewCell.swift
+//  F1
+//
+//  Created by Kaitlyn Sookdhew on 2024/04/08.
+//
+
+import UIKit
+
+class ConstructorTableViewCell: UITableViewCell {
+    
+    @IBOutlet weak private var round: UILabel!
+    @IBOutlet weak private var race: UILabel!
+    @IBOutlet weak private var firstDriver: UILabel!
+    @IBOutlet weak private var firstPoints: UILabel!
+    @IBOutlet weak private var firstPosition: UILabel!
+    @IBOutlet weak private var secondDriver: UILabel!
+    @IBOutlet weak private var secondPoints: UILabel!
+    @IBOutlet weak private var secondPosition: UILabel!
+    
+    static let identifier = Identifiers.constructorTableViewCell
```

Comment:
can add this to constants and remove static let identifier = Identifiers.constructorTableViewCell